<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calibracao</class>
 <widget class="QDialog" name="Calibracao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Alinhamento acima</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSlider" name="cimaSlider">
         <property name="maximum">
          <number>120</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cimaSpin">
         <property name="maximum">
          <number>120</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Alinhamento abaixo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="baixoSlider">
         <property name="minimum">
          <number>121</number>
         </property>
         <property name="maximum">
          <number>240</number>
         </property>
         <property name="value">
          <number>140</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="baixoSpin">
         <property name="minimum">
          <number>121</number>
         </property>
         <property name="maximum">
          <number>204</number>
         </property>
         <property name="value">
          <number>140</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Alinhamento direita</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSlider" name="direitaSlider">
         <property name="maximum">
          <number>160</number>
         </property>
         <property name="value">
          <number>140</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>true</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="direitaSpin">
         <property name="maximum">
          <number>160</number>
         </property>
         <property name="value">
          <number>140</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Alinhamento esquerda</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QSlider" name="esquerdaSlider">
         <property name="minimum">
          <number>161</number>
         </property>
         <property name="maximum">
          <number>320</number>
         </property>
         <property name="value">
          <number>180</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="esquerdaSpin">
         <property name="minimum">
          <number>161</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="value">
          <number>180</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cimaSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cimaSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cimaSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cimaSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>baixoSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>baixoSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>baixoSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>baixoSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>direitaSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>direitaSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>direitaSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>direitaSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>esquerdaSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>esquerdaSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>esquerdaSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>esquerdaSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
